crates/api_common/src/site.rs

```diff
@@ -163,6 +164,7 @@ pub struct GetSiteResponse {
   pub version: String,
   pub my_user: Option<MyUserInfo>,
```

Comment:
The front end will have to know which are your discussion languages, so `MyUserInfo` should contain a vector of your discussion_languages.